examples/mixture-mpi.py

```diff
@@ -378,12 +404,16 @@ def my_post_step(step, t, dt, state):
             logmgr.tick_after()
         return make_obj_array([fluid_state.cv, tseed]), dt
 
+    def dummy_rhs(t, state):
+        return make_obj_array([0*state[0], 0*state[1]])
+
```

Comment:
Looks like the `dummy_*` functions are unused here? (I see the ones in `combozzle-mpi` being used but not these.)